tests/adapter/dbt/tests/adapter/incremental/test_incremental_merge_exclude_columns.py

```diff
@@ -0,0 +1,127 @@
+import pytest
+from dbt.tests.util import run_dbt, check_relations_equal
+from collections import namedtuple
+
+
+models__merge_exclude_columns_sql = """
+{{ config(
+    materialized = 'incremental',
+    unique_key = 'id',
+    incremental_strategy='merge',
+    merge_exclude_columns=['msg']
+) }}
+
+{% if not is_incremental() %}
+
+select 1 as id, 'hello' as msg, 'blue' as color
+union all
+select 2 as id, 'goodbye' as msg, 'red' as color
+
+{% else %}
+
+select 1 as id, 'hey' as msg, 'blue' as color
```

Comment:
Re the above comment: I think ideally it'd be nice to label what's happening; that these rows are to overwrite the exist rows (except the excluded column). Just something in the naming or commenting to make intent clear.

Reply:
certainly can add a comment!